test/runtests.jl

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env julia
 
+if VERSION > v"0.5.0-dev"
+    Pkg.checkout("Optim")
+    Pkg.checkout("DataFrames")
+    Pkg.rm("GaussianMixtures")
+    Pkg.clone("https://github.com/jeff-regier/GaussianMixtures.jl.git")
```

Comment:
These run-time changes to which packages are installed seem like a bad idea. Better to document in the README the requirements for running on v0.5.